gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -149,6 +150,18 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         .getAddFeatureButtonClicks()
         .as(autoDisposable(this))
         .subscribe(viewModel::onAddFeatureButtonClick);
+    mapContainerViewModel
+        .getAddPolygonPointButtonClicks()
+        .as(autoDisposable(this))
+        .subscribe(mapContainerViewModel::onAddPolygonPointButtonClick);
+    mapContainerViewModel
+        .getSavePolygonRequest()
+        .as(autoDisposable(this))
+        .subscribe(mapContainerViewModel::onSavePolygonFeatureButtonClick);
+    mapContainerViewModel
+        .getRemoveLastVertexRequests()
+        .as(autoDisposable(this))
+        .subscribe(mapContainerViewModel::removeLastVertex);
```

Comment:
Same here - call VM directly from layout.

Reply:
updated and moved to layout